modules/core/client/views/careers.client.view.html

```diff
@@ -14,21 +14,28 @@ <h5 class="header col s12 light">Available Positions</h5>
 
 	</div>
 
-  <div class="row left">
-    <ul class="left jobsList">
-      <li class="jobList-item" ng-repeat="job in Jobs">
-        <div class="row">
-          <h5 class="col s12 light">{{job}}</h5>
-        </div>
-        <div class="row">
-          <span class="col s12 light">Description of Job goes here blah blah blah blah blah b lbkajasdfkjhasdkjfhaskjdhfksajdhfaksjdhfkajshfdksajhdfajksdfhasjkdfhajksdhfsdjkfhajkshdf
-                                      blah blah b lbkajasdfkjhasdkjfhaskjdhfksajdhfaksjdhfkajshfdksajhdfajksdfhasjkdfhajksdhfsdjkfhajkshdf
-                                      blah blah b lbkajasdfkjhasdkjfhaskjdhfksajdhfaksjdhfkajshfdksajhdfajksdfhasjkdfhajksdhfsdjkfhajkshdf....</span>
-        </div>
-        <div class="row">
-          <span class="right col s1 light">
-            <a id="contact-us-button" class="btn-large waves-effect waves-light orange" ng-click="openCareerApplication();">Apply Now</a>
-          </span>
+
+<!--   Search Bar  -->
+<nav>
+  <div class="nav-wrapper">
+    <form>
+      <div class="input-field">
+        <input id="search" class="grey" type="search" required>
+        <label for "search"><i class="material-icons">search</i></label>
+        <i class="material-icons">close</i>
+      </div>
+    </form>
+  </div>
+</nav>
+
+<!--   Job List  -->
+  <div class="row">
+    <ul class="jobsList">
+      <li class="jobList-item" data-ng-repeat="job in Jobs">
+        <div class="collapsible-header" ng-click="showBody1 === 'false' ? showBody1 = 'true' : showBody1 = 'false'"><i class="material-icons">games</i>{{job.title}}</div>
+        <div class=" collapsible-body" style="display: block" ng-show="showBody1 === 'true'"><p>Job Description: {{job.content}}</p>
+          <p>Creation Date: {{job.created}}</p>
+          <a id="contact-us-button" class="btn waves-effect waves-light blue" ng-click="openCareerApplication();">Apply Now</a>
         </div>
```

Comment:
get rid of the 'blue' class so that it styles this button like the rest of our buttons